Mapbox/MapboxMapsGestures/GestureManager.swift

```diff
@@ -283,7 +283,10 @@ extension GestureManager: GestureHandlerDelegate {
 
     // Pan has ended on the MapView with a residual `offset`
     internal func panEnded(with offset: CGPoint) {
-        self.cameraManager.moveCamera(by: offset, animated: true)
+        if let pitch = self.cameraManager.mapView?.pitch,
+           pitch == 0.0 {
```

Comment:
```suggestion
        if cameraManager.mapView?.pitch == 0.0 {
```

Reply:
I thought about this, but what if mapView is nil for some reason.

Reply:
If mapView is nil, the expression will evaluate to false.

Reply:
Which seems fine. You'll just be avoiding the assertionFailure inside of moveCamera

Reply:
This has the same behavior as what you wrote initially, it's just more concise.